lib/fluent/plugin/filter_kubernetes_metadata.rb

```diff
@@ -161,25 +170,34 @@ def configure(conf)
           end
         end
       end
+      if @use_journal
+        @merge_json_log_key = 'MESSAGE'
+        def self.filter_stream(tag, es)
```

Comment:
Actually I've just found out about [`alias_method`](http://ruby-doc.org/core-2.2.2/Module.html#method-i-alias_method) which seems a nice alternative to this dynamic def.